recipes/mvapich2/LICENSE

```diff
@@ -0,0 +1,46 @@
+                         MVAPICH COPYRIGHT
```

Comment:
The license file doesn't need to be copied into the recipe if it's in the package's tarball

Reply:
OK